Felienne/hedy
# Implement developers mode

Provide a general summary of your changes in the Title of the PR

**Description**

In this PR we implement a developers mode to give users the option to hide all "learning" elements that they don't need when focussing just on coding. By pressing the `toggle()` the commands and adventure tabs will be hidden and the code and output window will be increased in width and height.

**Fix for**

This PR fixed #1276.

**How to test**

Press the toggle() button below the "run code" button and notice that all non-coding elements of the page will disappear. Press it again to return to the original view. Notice that the toggle will not be remembered when switching levels.

**Screenshots**
![Schermafbeelding 2021-11-15 om 14 05 24](https://user-images.githubusercontent.com/38155424/141786900-5b356330-53d4-4422-8de4-3595ee780ad0.png)
![Schermafbeelding 2021-11-15 om 14 05 32](https://user-images.githubusercontent.com/38155424/141786908-bf5b5826-cbfc-4b8b-bcfa-a40eba1ece9d.png)

**Checklist**

If you're unsure about any of these, don't hesitate to ask. We're here to help!
